docs/source/markdown/options/network.md

```diff
@@ -62,11 +60,20 @@ Valid _mode_ values are:
     container to host using the gateway address. The latter can be overridden
     by passing **--map-gw** in the pasta-specific options (despite not being an
     actual pasta(1) option). \
+    For better integration with DNS handling **--dns-forward 169.254.1.1** is passed
+    and the ip is added to resolv.conf as first entry. It is possible to pass **--dns-forward**
+    explicitly in case a different ip should be used.
+    To make the `host.containers.internal` /etc/hosts entry work and allow connections
+    to the host **--map-guest-addr 169.254.1.2** is passed as well, again it can be set
+    explicitly to choose a different ip. \
     Also, **-t none** and **-u none** are passed if, respectively, no TCP or
-    UDP port forwarding from host to container is configured, to disable
-    automatic port forwarding based on bound ports. Similarly, **-T none** and
-    **-U none** are given to disable the same functionality from container to
+    UDP port forwarding from host to container is configured (i.e. via Podman's
```

Comment:
I would drop "i.e.", that's a pretty direct relationship between those options and the configuration of port forwarding.